workflow-examples/run_examples.sh

```diff
@@ -335,6 +335,67 @@ run_escalation_flow() {
   echo "                                                "
 }
 
+run_prebuilt_flow() {
```

Comment:
What do you think about using sdk to create this flow?

/cc @gciavarrini

Reply:
I agree its can be done using the sdk
Does the "java SDK" solution ready to replace the bash script?
Anyway, would like to have this in different PR

Reply:
Created the Java test file instead the bash
@gciavarrini please take a look